gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -59,7 +59,7 @@ module.exports = function(env) {
     webpackConfig.output.pathinfo = true
 
     // Create new entries object with webpack-hot-middleware and react-hot-loader (if enabled)
-    if(TASK_CONFIG.javascripts.hot.enabled !== false) {
+    if(TASK_CONFIG.javascripts.hot && TASK_CONFIG.javascripts.hot.enabled !== false) {
```

Comment:
Oops. I think this should actually be: 

```js
if(!TASK_CONFIG.javascripts.hot || TASK_CONFIG.javascripts.hot.enabled !== false`)
```

Basically I want hot-loading enabled by default, unless you specifically turn it off. So if there's no config for it, assume it's on.